src/Periphery/LiFiDEXAggregator.sol

```diff
@@ -750,6 +765,45 @@ contract LiFiDEXAggregator is WithdrawablePeriphery {
             }
         }
     }
+
+    function swapVelodromeV2(
+        uint256 stream,
+        address from,
+        address tokenIn,
+        uint256 amountIn
+    ) private {
+        address pool = stream.readAddress();
+        uint8 direction = stream.readUint8();
+        address to = stream.readAddress();
+        stream.readUint24(); // pool fee in 1/1_000_000
+        stream.readUint8() > 0;
+        bool callback = stream.readUint8() > 0; // if true then run callback after swap with tokenIn as flashloan data. Will revert if contract (to) does not impelent IVelodromeV2PoolCallee
+        // we don't need 'fee' and 'stable' flags since the pool handles that internally
```

Comment:
would propose the following update: 

```solidity
// we don't need 'pool fee' and 'stable' flag since the pool handles that internally but (must) read them anyway to forward to pointer of the stream
stream.readUint24(); // read pool fee (in 1/1_000_000)
stream.readUint8(); // read stable flag 
```


note: I removed the `> 0` as this is useless and a waste of gas